src/interface/meshblock_data.hpp

```diff
@@ -148,18 +146,20 @@ class MeshBlockData {
                              "Couldn't find variable '" + label + "'");
     return it->second;
   }
+  std::shared_ptr<CellVariable<T>> GetCellVarPtr(const Uid_t &uid) const {
```

Comment:
Maybe a `PARTHENON_DEBUG_THROWS(varUidMap_.count(uid), "Variable ID not found!");`?

Reply:
Just realized this isn't necessary because `map.at(val)` has bounds checking. I added it anyway though for a nicer error message.